app/controllers/users_controller.rb

```diff
@@ -0,0 +1,71 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[ show edit update destroy ]
+  before_action :authenticate_user!, :only => [:edit, :destroy, :update]
+
+  # GET /users or /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1 or /users/1.json
+  def show
+  end
+
+  # GET /users/new
+  def new
+    @user = User.new
+  end
+
+  # GET /users/1/edit
+  def edit
+  end
+
+  # POST /users or /users.json
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
```

Comment:
Why do we respond to html and json?

Reply:
I used the scaffolding method, i wrote it on Trello! Not sure if i can leave the controller like that, should I write it by myself? By now, I think i can just delete format.json - we don't need potential javascript for client, right?

Reply:
As i understand, render_to do format.html is a method similar to flash notifications that were introduced in a course, so i would leave format.html and delete format.json as they are not nessesary :D

Reply:
Yeah I would remove the json format for now.

Reply:
Thanks Peter